README.md

```diff
@@ -59,7 +61,7 @@ I could really use your support in giving me a star on GitHub, recommending feat
 
 Since I am no longer a data scientist and JupyterLab has stopped being part of my main workflow, I am unable to commit time to work on this project due to other commitments.
 
-----
+---
 
 ## Contributors
 
```

Comment:
@ryantam626 is this addition ok? Would you like to add a link here?

```suggestion
## Contributors

This extension was originally developed and maintained by @ryantam626.
```

Reply:
of course! i guess a link to my github profile?

Reply:
:+1: updated to include a link